core/storage/sqlite3_ondisk.rs

```diff
@@ -1167,6 +1188,42 @@ pub fn payload_overflows(
     (true, space_left + 4)
 }
 
+pub fn checksum_wal(
+    buf: &[u8],
+    wal_header: &WalHeader,
+    input: (u32, u32),
+    native_endian: bool, // Sqlite interprets big endian as "native"
```

Comment:
`wal_matches_native_endianness` or something?

Reply:
in this case I don't agree, it just means to read bytes in native endian order or not. Not relevant to WAL.